### DISCUSSION TITLE
Drop off location (user with only upload permissions)

### DISCUSSION THREAD

**Discussion by nfaxi1** [drakkan/sftpgo#1187] [Q&A]
I tried creating a user with only upload permissions to build a public drop off point where everyone is using the same user but cannot see each others uploads. Unfortunately  I get an error after loggin in with that user "Failed to get directory listing: Unable to get directory contents". This would be fine but it does not let me upload files now.
Anything I am missing or any way to get around this?

**Comment by drakkan** [comment]
I think a write only share is more appropriate for your use case

![Schermata del 2023-02-11 12-36-46](https://user-images.githubusercontent.com/553263/218255972-ffd11818-3416-43c7-9c7a-b070ea788ba4.png)

**Reply by nfaxi1** [reply]
Sounds exactly like what I was looking for, how do I accomplish this with sftpgo? Or did you mean I should implement it without it?

**Reply by drakkan** [reply]
> Sounds exactly like what I was looking for, how do I accomplish this with sftpgo? Or did you mean I should implement it without it?

the screen above is an SFTPGo write only share.

**Reply by nfaxi1** [reply]
That's what I thought but in the admin view under "Folders" I can only create normal shares. How would I reproduce your suggested write only share? The screenshot does not really show that unfortunately.

**Reply by drakkan** [reply]
> That's what I thought but in the admin view under "Folders" I can only create normal shares. How would I reproduce your suggested write only share? The screenshot does not really show that unfortunately.

I no longer provide step by step support, sorry

https://github.com/drakkan/sftpgo#support-policy
